README.md

```diff
@@ -84,114 +53,100 @@ To remove a dependency from your project, you can use the `poetry remove` comman
+## Use Poe the Poet as Task Runner
 
 [Poe the Poet](https://poethepoet.natn.io/index.html) is a task runner that simplifies running common tasks in a Python project.
 
-To have the command availalbe as `poetry poe <command>` as seen in the following examples, you need to install Poe as a plugin to poetry:
+To have the command available as `poetry poe <command>` as seen in the following examples, you need to install poe as a plugin to Poetry:
 
 ```bash
 poetry self add 'poethepoet[poetry_plugin]'
 ```
 
-If this plugin is not installed you may have to run Poe as a script within the Poetry environment using `poetry run poe <command>`.
-
-## Usage
-
-> Explain how to use the project. You can create multiple subsections (H3). Include the instructions or provide links to
-> the related documentation.
+If this plugin is not installed, you may have to run poe as a script within the Poetry environment using `poetry run poe <command>`.
 
 ## Development
 
-This sections describes how to develop and run the kyma-companion fastapi app.
-
-### Redis Server
+### Redis Server 
 
-Before running the application, the Redis server must be provided that is used to store the conversation with a LLM.
-Therefore, REDIS_URL has to be provided as an environment variable.
+Before running the application, the Redis server must be provided. It is used to store the conversation with a large language model (LLM).
+Therefore, **REDIS_URL** must be provided as an environment variable.
 
-Here is the [documentation](https://github.tools.sap/kyma/ai-force/blob/main/docs/infrastructure/setup.md#15-redis) how to create a Redis server. For example,
-`REDIS_URL="redis://{host or ip}:6379"`
+Here is the [documentation](https://github.tools.sap/kyma/ai-force/blob/main/docs/infrastructure/setup.md#15-redis) <!--the link must be replaced when the OS documentation is available --> on how to create a Redis server. 
+For example, `REDIS_URL="redis://{host or ip}:6379"`
 
-### Running
+### Running Kyma Companion Locally
 
-You can start the FastApi application with the following options:
+You can start the FastAPI application with the following options:
 
 ```bash
 poetry run fastapi dev src/main.py --port 8000
 ```
 
-or, directly with FastApi:
-
-```bash
-fastapi dev src/main.py --port 8000
-```
-
-or, with a poe task:
+Or, with a poe task:
 
 ```bash
 $ poetry poe run-local
 ```
 
-Prefer running it with poetry as it activates and uses its virtual environment if not activated yet.  
-**NOTE:** it cannot be run with python directly.
+It is recommended to run Kyma Companion with Poetry as it activates and uses its virtual environment if not activated yet.
+
+> [!NOTE] You cannot run it with Python directly.
 
 ### Debugging
 
-The FastAPI application can be debugged with PyCharm or VS Code. Follow the following guidelines for these IDEs:
+As the companion uses the FastAPI framework, refer to the following documentation on how to debug the application with the respective IDE:
```

Comment:
```suggestion
Because the companion uses the FastAPI framework, read the following documentation on how to debug the application with the respective IDE:
```